### PR TITLE
Adds bluespace bananas

### PR DESCRIPTION
**Pull request by Sestren413**
Inspired by an accidental misstatement, I created bluespace bananas. The banana itself has singulo in it, like the bluespace tomato. The resulting peel has the same teleportation radius as a bluespace tomato of the same potency. Both items are bluespace 3 tech like the tomato.

I added blue versions of the existing banana icons. Tried to select colors to resemble the bluespace tomato plants. I honestly think they look a bit shit but I am not respriting bananas.

~~The plant was not added to the mutation list for regular banana plants. I do not want to unleash a horde of teleporting clown pranksters on the station without admin consent.~~

:cl:
add: Botanists have access to a new banana mutation now, bluespace bananas. These will teleport anyone unfortunate enough to slip on their peel.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
So like teleprods, but a banana peel?
I like it.

**Comment by duncathan** [comment]
This is a terrible, terrible idea
:+1:

**Comment by GunHog** [comment]
Bag of Holding while slipping on one of these....you will never be seen again! :+1:

**Comment by Alek2ander** [comment]
Needs some kind of sparks on teleport to indicate that it IS a teleport. And probably a sound.

**Comment by Sestren413** [comment]
When I compiled it, it looked like do_teleport function has the sparking and noise built in, because it was doing it for me the same as the tomatoes do.

**Comment by bgobandit** [comment]
daisy intensifies

File: code/modules/hydroponics/growninedible.dm [hunk 1]
```diff
@@ -284,6 +284,19 @@
...
+	desc = "A peel from a bluespace banana."
+	icon = 'icons/obj/hydroponics/harvest.dmi'
+	icon_state = "banana_peel_blue"
+	origin_tech = "bluespace=3"
+
+/obj/item/weapon/grown/bananapeel/bluespace/Crossed(AM as mob|obj)
+	if(..())
+		var/teleport_radius = potency / 10
+		do_teleport(AM, get_turf(AM), teleport_radius)
+		qdel(src)    //let's not make one peel infinitely reusable
```
Comment (ChangelingRain) [inline code/modules/hydroponics/growninedible.dm]: What fun-ruining is this? Just have the peel teleport too.

Comment (PKPenguin321) [inline code/modules/hydroponics/growninedible.dm]: give it like a prob50 of deleting or teleporting

**Comment by Core0verload** [comment]
Why no "You slip thought space-time!" message?

**Comment by newfren** [comment]
Only like 3 people ever actually mutate plants anyway and one of them is reed glover who is above reproach make it a banana mutation.

**Comment by tkdrg** [comment]
Just make it a mutation honestly. If it ends up being awful we can always make it admin-only.

**Comment by PKPenguin321** [comment]
yeah, just make it a mutation of bananas. it'll be fine

**Comment by newfren** [comment]
Plus if it's a banana mutation you have to fight against mimanas so it takes more time + mutagen to roll into.

**Comment by Chocobro** [comment]
@Sestren413 this is a great idea but in future please ask the king of hydroponics first thank

**Comment by nerdinexile** [comment]
>king of hydroponics

**Comment by Chocobro** [comment]
considering i came up with 7/8 of /tg/station's plants
yeah
i am

**Comment by duncathan** [comment]
muh codebabby

**Comment by Chocobro** [comment]
spritebabby, duncan of the yogscast

**Pull request merged**